src/client/localplayer.cpp

```diff
@@ -170,8 +170,8 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 		std::vector<CollisionInfo> *collision_info)
 {
 	if (!collision_info || collision_info->empty()) {
-		// Node below the feet, update each ClientEnvironment::step()
-		m_standing_node = floatToInt(m_position, BS) - v3s16(0, 1, 0);
+		// Node at feet position, update each ClientEnvironment::step()
+		m_standing_node = floatToInt(m_position, BS);
```

Comment:
why did this not cause problems before?

Reply:
Because
1) It's overwritten after the first of many `move()` calls inside `step()` as soon there's a collision
2) It doesn't matter in fly mode

However, now it has to be this way to properly detect nodes when the player is floating.

Reply:
@rubenwardy ok for you?